Makefile

```diff
@@ -0,0 +1,16 @@
+all: build
+
+APP_NAME=cluster-ingress-operator
+BIN=cluster-ingress-operator
+MAIN_PKG=github.com/openshift/$(APP_NAME)/cmd/cluster-ingress-operator
```

Comment:
Should that be `MAIN_PKG=github.com/openshift/$(APP_NAME)/cmd/$(APP_NAME)`? Or is it useful to parameterize `APP_NAME` at all?